### ISSUE TITLE
License: GPL too restrictive?

### ISSUE THREAD

**Issue by aksdb** [ousia/from-pandoc-to-context#17]
I want to bring to attention, that the GPL might be the wrong license here.
GPL extends to "derived work". Since I cannot use pandoc-xhtml without also creating a proper layout around it (as you do with your `from-pandoc-to-context.tex`), I am kinda forced to derive it.
To my understanding, I would now also have to release my custom layout, not just my modifications to pandoc-xhtml, which is, imho, a huge problem.
Maybe the LGPL or something completely different is better suited?

**Comment by ousia** [comment]
@aksdb, thanks for noticing the point, but I’d rather avoid changing the license.

_Bearbeitungen_ are “derivative works” or simply “derivatives” (sorry, this seems to be legal terminology).

Although I’m not a lawyer myself (and this isn’t legal advice), mere aggregation may not constitute a derivative work.

I have to take a look at it when I have time. But I guess the same issue would arise when you use ConTeXt (and explicitly with `\usemodule`).

**Comment by aksdb** [comment]
@ousia Yes, the ConTeXt license may be problematic too, if someone starts to challenge someone into releasing the source.

Anyway, I compare modules and environment with software libraries (DLLs, Shared Objects). They are used via a defined interface. The LGPL is meant for exactly cases like this. If I only intend to use the interface, I'm fine to do so in whichever way I want. If I change the inner workings of the library in any way (adding stuff, removing stuff, changing stuff) I have to distribute the changes together with my version of that library. I guess this is more or less what you have in mind for pandoc-xhtml.tex.

The GPL on the other hand enforced everyone who somehow profits from the work to be licensed under a compatible license (which usually is also GPL). So if my program code uses a GPL-licensed library via its defined interface, I'm still forced to also release the code of my whole program, not just changed I made to the library.

All in all it isn't anything urgent, since you probably won't start to sue people for somehow using pandoc-xhtml ;-) It was just something I wanted to bring to your attention.

**Comment by ousia** [comment]
@aksdb, I think that the basic premise is simple:

> All files contained in the repository `https://github.com/ousia/from-pandoc-to-context.git` cannot be relicensed either unmodified or not.
